components/forms/RadioButton/react.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { JSDOM } from 'jsdom'
+import RadioButton from './react'
+
+const jsdom = new JSDOM('<!doctype html><html><body><div id="app">Mount here</div></body></html>')
+const { window } = jsdom
+
+describe('RadioButton', () => {
+  let props
+  let wrapper
+
+  beforeEach(() => {
+    props = {
+      checked: false,
+      onChange: jest.fn(),
+      value: 'true',
+    }
+
+    wrapper = mount(<RadioButton {...props} />, {
+      attachTo: window.document.querySelector('#app'),
+    })
+  })
+
+  describe('with snapshots', () => {
+    it('should render correctly', () => {
+      expect(wrapper).toMatchSnapshot()
+    })
+  })
+
+  describe('with user actions', () => {
```

Comment:
It's great that the click event is tested. We should also test for the following:

-if a key is pressed, that the onChange event is fired
-when the component renders with the checked property passed in as true, that the radio button is check
-when the component renders with no checked property, that the check does not render
-when the component renders with a check property passed in as any value (aside from true or false), that the condition is handled. I believe it would render checked true in this case but double check me on that.
-when the component renders with the check property passed in as false, that the checked attribute does not show (again, double check me on this).
-Check that the value passed in shows up as the value of the radio button